fanuc_r2000ic_support/urdf/r2000ic210l_macro.xacro

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find fanuc_resources)/urdf/common_materials.xacro"/>
```

Comment:
please move this to be inside the scope of the `xacro:macro` (similar to how I moved it in the CRX-10iA/L file).